README.md

```diff
@@ -90,6 +90,7 @@ container.
         // Global variables passed to twig templates
         'ga_tracking' => 'UA-XXXXX-X'
     ],
+    'timezone' => 'default timezone identifier, e.g.: America/New_York'
```

Comment:
Uhm yeah, you are right. Looking at the expressive docs, both are being used there as well.

@weierophinney What would be the preferred way?